CMakeLists.txt

```diff
@@ -366,6 +366,11 @@ if (NOT osal STREQUAL "embedded" )
   set ( LIBFLUID_LIBS Threads::Threads )
 endif()
 
+# Add -llog if we compile for Android
+if (ANDROID_ABI)
+  set ( CMAKE_SHARED_LINKER_FLAGS "-Wl,-llog ${CMAKE_SHARED_LINKER_FLAGS}" )
```

Comment:
Why are you using `CMAKE_SHARED_LINKER_FLAGS` here? I would have expected a call to either `target_link_libraries` or `target_link_options` as we do it e.g. here: https://github.com/FluidSynth/fluidsynth/blob/9f42d51cae0b6e11be6c184824caf1ccec433342/src/CMakeLists.txt#L367

Specifying  a public link dependency should correctly expose it in the auto-generated `FluidSynthTargets.cmake`, letting others know that the compiled fluidsynth binary are no longer self contained as before, but now require a log library from the NDK.

Usually, I'd also expose `-llog` in the pkgconfig file, but I guess this isn't used for Android.